.circleci/config.yml

```diff
@@ -23,63 +23,23 @@ jobs:
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
-      - run: mvn package dependency:go-offline
+      #- run: mvn package dependency:go-offline
 
       - save_cache:
           paths:
             - ~/.m2
           key: v1-dependencies-{{ checksum "pom.xml" }}
 
-      # run tests!
```

Comment:
Some comments start with capital letter and some don't. If you want nitpick mode I'd say be consistent :)